src/db_state.rs

```diff
@@ -388,6 +395,8 @@ impl DbState {
                 next_tick: memtable_tick,
             });
         }
+        // update the persisted manifest last_l0_seq as the latest seq
+        state.core.last_l0_seq = self.last_seq;
```

Comment:
This doesn't seem right. IIUC, `last_seq` represents the sequence number for the last write that went through the WAL. It's incremented whenever we flush a WAL write:

https://github.com/slatedb/slatedb/pull/389/files#diff-51529babecd6e4018b78a49622df5b5d2c480476389ec07c0b4943d8bdbaa35eR60

Wouldn't setting `last_l0_seq` to `last_seq` here result in a sequence number _higher_ than the max seqnum in the immutable memtable (if we've had more WAL writes)? I think we might need a `imm_memtable.last_seq()` to get the true last sequence number in the immutable memtable.

Reply:
+1 - we should be tracking the highest seq num in each L0 SST

Reply:
sorry it's a bug, fixed in fbada6c, PTAL